docs/src/main/docs/about/01_overview.adoc

```diff
@@ -0,0 +1,103 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Overview
+:description: Helidon documentation
+:keywords: helidon, java, microservices, microprofile, documentation
+
+== Get going
+
+[PILLARS]
+====
+[CARD]
+.Getting Started
+[icon=assistant,link=getting-started/02_base-example.adoc]
+--
+Quickly create your first application under 2 minutes.
+--
+
+[CARD]
+.Guides
+[icon=explore,link=guides/01_overview.adoc]
+--
+Hands-on guides.
+--
+====
+
+== Components
+
+[PILLARS]
+====
+[CARD]
+.WebServer
+[icon=settings_ethernet,link=webserver/01_introduction.adoc]
+--
+A programmatic HTTP API with reactive features, powered by Netty.
+--
+
+[CARD]
+.Config
+[icon=settings,link=config/01_introduction.adoc]
+--
+A flexible configuration framework with support for multiple sources and
+ formats.
+--
+
+[CARD]
+.Security
+[icon=security,link=security/01_introduction.adoc]
+--
+A tool-chain to handle authentication, authorization and context propagation.
+--
+
+[CARD]
+.Microprofile
+[icon=widgets,link=microprofile/01_introduction.adoc]
+--
+MicroProfile is a platform definition that is familiar to enterprise Java
+ developers.
+--
```

Comment:
A platform definition that is familiar to enterprise Java developers.

(for consistency with the other descriptions)